velox/type/tests/ConversionsTest.cpp

```diff
@@ -1012,14 +1012,14 @@ TEST_F(ConversionsTest, toTimestamp) {
             "2000-01-01",
             "1970-01-01 00:00:00",
             "2000-01-01 12:21:56",
-            "1970-01-01 00:00:00-02:00",
```

Comment:
That's right. The problem is that you cannot convert strings into a timestamp directly with the Converter api since you don't know the timezone. So in CastHooks we can check the session parameter and do the appropriate conversion.